805-airgap.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Disconnected Scenario
+weight: 805
+---
+
+# Disconnected Scenarios
+
+This section is non-normative and describes the disconnected (air gapped) scenario mentioned in the introduction of [CNAB-Core](100-CNAB.md).
+
+## Summary
+
+Using CNAB in a disconnected environment involves transferring a bundle and its images into the 
+environment such that the bundle can be installed and the bundled software executed successfully. 
+
+## Internet Access
+
+A typical disconnected scenario will have limited, intermittent or no internet access, whether by design or by situation.
+To install a bundle directly in a disconnected environment, the bundle and its images need to be included in a CNAB Thick Bundle
```

Comment:
Link to the thick bundle definition here?

Reply:
Fixed.